flocker/node/script.py

```diff
@@ -552,8 +404,6 @@ def get_api(backend, api_args, reactor, cluster_id):
 
 class AgentService(PClass):
     """
-    :ivar backends: ``BackendDescription`` instances describing how to use each
```

Comment:
The original refactoring didn't have an attribute here. I forgot to add back the documentation when I added it back.